packages/ckeditor5-image/src/imageinsert/imageinsertui.ts

```diff
@@ -83,7 +81,7 @@ export default class ImageInsertUI extends Plugin {
 		} );
 
 		if ( uploadImageCommand ) {
-			const splitButtonView = this.dropdownView.buttonView;
+			const splitButtonView = this.dropdownView.buttonView as any;
```

Comment:
Please avoid `any`.

Reply:
I can't eliminate `as any` here. There's an assignment to `actionView` which is `readonly` field.